build.gradle

```diff
@@ -141,9 +153,11 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.9'
     implementation 'org.apache.commons:commons-text:1.8'
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.10.2'
+    implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.10.2'
 
```

Comment:
If we're switching to gson can we remove the jackson libs?

Reply:
I'm not suggesting switching to gson for the spring boot api calls, just for the manual hydration in this class.  Our spring boot apps in all projects use jackson so would be worth keeping it that way.    The case notes to probation service follows the same pattern in using gson just for manual hydration.

Reply:
See https://www.callicoder.com/configuring-spring-boot-to-use-gson-instead-of-jackson/ for more information on integrating gson with spring boot